rag-langchain/rag_app.py

```diff
@@ -18,15 +18,22 @@
 import argparse
 import pathlib
 
+model_service = os.getenv("MODEL_SERVICE_ENDPOINT",
+                          "http://0.0.0.0:8001/v1")
+
 parser = argparse.ArgumentParser()
 parser.add_argument("-c", "--chunk_size", default=150)
 parser.add_argument("-e", "--embedding_model", default="BAAI/bge-base-en-v1.5")
 parser.add_argument("-H", "--vdb_host", default="0.0.0.0")
 parser.add_argument("-p", "--vdb_port", default="8000")
 parser.add_argument("-n", "--name", default="test_collection")
-parser.add_argument("-m", "--model_url", default="http://0.0.0.0:8001/v1")
+parser.add_argument("-m", "--model_url", default=model_service)
```

Comment:
I think this line can actually just be removed. Its duplicate information as `model_service`